core/src/modules/configUser.ts

```diff
@@ -26,7 +26,7 @@ export namespace ConfigUser {
 
   async function subscribe(email: string, subscribed: boolean = true) {
     if (!subscribed) return null;
-    await GrouparooSubscription({ email, subscribed });
+    return GrouparooSubscription({ email, subscribed });
```

Comment:
You can un-async this function with this change.

Reply:
Unless you're expecting it to throw, in which case this should be `return await ...`

Reply:
I think I'm going to revert this - it might throw